zuul.d/adoption.yaml

```diff
@@ -84,6 +91,9 @@
               tenant:
                 ip: 172.19.0.100
                 config_nm: false
+              storage_mgmt:
```

Comment:
so there must be some os-net-config configuration involved on the rdo-jobs side too?

Reply:
It should be handled by the dataplane operator, that based on [this](https://github.com/openstack-k8s-operators/ci-framework/blob/main/hooks/playbooks/fetch_compute_facts.yml#L184) generates and runs os-net-config.

Reply:
@marios I don't think so, the  standalone looks fine, it seems it's only missing on the crc side
@pablintino thanks for the pointer, I have to dig a bit more into this, in the adoption job we don't use deploy EDPM using the framework

Reply:
@cescgina++ !

so the new overcloud deploy job hit this issue/need yesterday @cescgina there https://logserver.rdoproject.org/74/51274/11/check/data-plane-adoption-osp17-multinode-to-extracted-crc/35aead8/controller/ci-framework-data/logs/ci_script_010_run_tripleo_deploy.log

```
 2024-03-13 09:39:34.206366 | fa163e81-6f46-45f9-c884-000000001217 |      FATAL | Check IP responsiveness | np0004536286 | error={"attempts": 10, "changed": false, "cmd": ["ping", "-w", "10", "-c", "5", "172.20.0.103"], "delta": "0:00:03.088312", "end": "2024-03-13 05:39:34.165463", "msg": "non-zero return code", "rc": 1, "start": "2024-03-13 05:39:31.077151", "stderr": "", "stderr_lines": [], "stdout": "PING 172.20.0.103 (172.20.0.103) 56(84) bytes of data.\nFrom 172.20.0.104 icmp_seq=1 Destination Host Unreachable\nFrom 172.20.0.104 icmp_seq=2 Destination Host Unreachable\nFrom 172.20.0.104 icmp_seq=3 Destination Host Unreachable\n\n--- 172.20.0.103 ping statistics ---\n3 packets transmitted, 0 received, +3 errors, 100% packet loss, time 2057ms\npipe 3", "stdout_lines": ["PING 172.20.0.103 (172.20.0.103) 56(84) bytes of data.", "From 172.20.0.104 icmp_seq=1 Destination Host Unreachable", "From 172.20.0.104 icmp_seq=2 Destination Host Unreachable", "From 172.20.0.104 icmp_seq=3 Destination Host Unreachable", "", "--- 172.20.0.103 ping statistics ---", "3 packets transmitted, 0 received, +3 errors, 100% packet loss, time 2057ms", "pipe 3"]}
```

I can ping all the other networks from the controllers but not 172.20.0.103 which is storage_mgmt. I did some digging and realised we don't have that on crc which is also acting as the dns

the controller sends out the arp request for it but no one is answering

```
 [root@tripleo-controller-2 zuul]# tcpdump -i vlan23
dropped privs to tcpdump
tcpdump: verbose output suppressed, use -v[v]... for full protocol decode
listening on vlan23, link-type EN10MB (Ethernet), snapshot length 262144 bytes
05:38:39.906919 ARP, Request who-has np0004536285.storagemgmt.localdomain tell np0004536287.storagemgmt.localdomain, length 28

```